lib/rules/sotd/pp.js

```diff
@@ -1,38 +1,41 @@
-const PowerPromise = require('promise');
-const w3capi = require('node-w3capi');
 const util = require('../../util');
 
 const self = {
     name: 'sotd.pp',
 };
 
-const NO_PP_TEXT =
-    'The 1 August 2017 W3C Patent Policy does not carry any licensing requirements or commitments on this document.';
+const ppLink = 'https://www.w3.org/Consortium/Patent-Policy/';
+const pp2020 = 'https://www.w3.org/Consortium/Patent-Policy-20200915/';
+const pp2017 = 'https://www.w3.org/Consortium/Patent-Policy-20170801/';
 
-function buildWanted(groups, sr, isIGDeliverable) {
+function buildWanted(groups, sr, publishedByWgOnly, ppLink) {
     const { config } = sr;
     let wanted;
     const result = {};
     const isPP2002 = config.patentPolicy === 'pp2002';
+    const isPP2017 = ppLink === pp2017;
     const isRecTrack = config.track === 'Recommendation';
     const isAmended = config.amended;
+
     if (isPP2002)
         wanted =
             'This document is governed by the 24 January 2002 CPP as amended by the W3C Patent ' +
             'Policy Transition Procedure\\. ';
     else {
+        let ppText = '( 15 September 2020)?';
+        if (isPP2017) ppText = ' 1 August 2017';
+
         const start = !isAmended
             ? 'This document '
             : 'The Recommendation from which this document is derived ';
         wanted = `${start}was produced by ${
             groups.length === 2 ? 'groups ' : 'a group '
-        }operating under the( 15 September 2020| 1 August 2017)? W3C Patent Policy\\. ?`;
-
-        if (!isRecTrack || isIGDeliverable) wanted = NO_PP_TEXT;
+        }operating under the${ppText} W3C Patent Policy\\. ?`;
 
-        if (config.informativeOnly && config.track !== 'Note')
-            wanted += 'This document is informative only\\. ';
+        if (!isRecTrack || !publishedByWgOnly)
```

Comment:
I don't think it's necessary to check if `publishedByWgOnly` is false here. All the documents that aren't on REC track will have that sentence.
This should also be integrated in the `if` at line 39 (`if...else`).

Also, I don't think that code works with Statements which have another sentence before the PP text. Not that this first sentence is being checked by `stability.js` so the rules are overlapping.